dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/factory/impl/BitstreamMetadataValueMovePatchOperation.java

```diff
@@ -35,12 +36,17 @@
     @Autowired
     ItemService itemService;
 
+    @Autowired
+    BitstreamMetadataValuePathUtils bitstreamMetadataValuePathUtils;
```

Comment:
When trying to test this PR, I got spring issues with this auto-wiring

Reply:
This still seems to be occurring in my local installation:
```
***************************
APPLICATION FAILED TO START
***************************

Description:

Field bitstreamMetadataValuePathUtils in org.dspace.app.rest.submit.factory.impl.BitstreamMetadataValueMovePatchOperation required a single bean, but 2 were found:
        - org.dspace.app.rest.utils.BitstreamMetadataValuePathUtils#0: a programmatically registered singleton  - org.dspace.app.rest.utils.BitstreamMetadataValuePathUtils#1: a programmatically registered singleton

Action:

Consider marking one of the beans as @Primary, updating the consumer to accept multiple beans, or using @Qualifier to identify the bean that should be consumed
```

Reply:
I think @benbosman may be hitting a problem here in how we are loading this bean.  See [this comment](https://github.com/DSpace/DSpace/pull/2760#discussion_r443732844) that I just added inline.

Reply:
This bug was resolved by the most recent changes 🎉